main.py

```
@@ -630,7 +630,10 @@ async def lounge_host(update: Update, context: ContextTypes.DEFAULT_TYPE):
                 invitations[str(member.id)] = inviter_name
                 save_state()
             try:
-                keyboard = [[InlineKeyboardButton("📖 Open Menu", callback_data="show_menu")]]
+                keyboard = [
+                    [InlineKeyboardButton("📖 Open Menu", callback_data="show_menu")],
```

Ruff (0.15.12)

[error] Local variable `InlineKeyboardButton` referenced before assignment

(F823)